main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -329,6 +329,9 @@ public void onCreate(final Bundle savedInstanceState) {
         gp = getArguments().getParcelable(GEOPOINT_ARG);
         if (gp == null) {
             gp = Sensors.getInstance().currentGeo().getCoords();
+            if (savedState != null) {
+                gp = new Geopoint(savedState.plainLat, savedState.plainLon);
+            }
```

Comment:
If savedState != null then you initialize gp twice.
Better e.g.:
~~~
 if (savedState != null) {
    gp = new Geopoint(savedState.plainLat, savedState.plainLon);
} else {
    gp = Sensors.getInstance().currentGeo().getCoords();
}
~~~

Reply:
suggestion was changed in last commit/push